src/command_modules/azure-cli-network/azure/cli/command_modules/network/mgmt_local_gateway/azuredeploy.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "localAddressPrefix": {
+      "type": "string",
+      "defaultValue": "192.168.0.0/16",
+      "metadata": {
+        "description": "CIDR block representing the address space of the OnPremise VPN network's Subnet."
+      }
+    },
+    "gatewayIpAddress": {
+      "type": "string",
+      "metadata": {
+        "description": "Gateway's Public IP address."
```

Comment:
this is the actual IP address.  I haven't found a good terminology to differentiate the two yet.  Perhaps Azure Public IP Address could be used for the object ID reference, but that case is more common, so finding a term for the actual IP Address would be better.  I think for now I will add an example into the description.

---
In reply to: [74152387](https://github.com/Azure/azure-cli/pull/668#discussion_r74152387) [](ancestors = 74152387)